polyfills.ts

```diff
@@ -1,20 +1,19 @@
 // Random values must be imported first for security.
+import "@ethersproject/shims";
```

Comment:
Did you do a quick sanity run?
I think sometimes the polyfills require a certain ordering to work, so may be better to actually ignore any ordering in this particular file

Reply:
No didn't do sanity run. I've put back the order and put a comment to not change the order just in case

Reply:
Second this 👍